src/en/howto-applicationupdateseries.md

```diff
@@ -0,0 +1,117 @@
+Title: Change an application's series pre-juju2.3
+
+#  Change an application's series pre-juju2.3
```

Comment:
I think "(prior to 2.3)" works slightly better

Reply:
updated